frontend/www/js/omegaup/course/edit.js

```diff
@@ -546,8 +581,21 @@ OmegaUp.on('ready', function() {
         },
       });
     },
-    data: {
-      students: [],
+    data: { students: [], data: [] },
+    methods: {
+      arbitrateRequest: (ev, username, resolution) => {
+        omegaup.API.Course.arbitrateRequest({
```

Comment:
```suggestion
        api.Course.arbitrateRequest({
```

siempre que puedas, hay que preferir `api` (de `api_transitional`) sobre `API`. y hay que preferir `API` sobre `omegaup.API`.

Reply:
relacionado: https://github.com/omegaup/omegaup/pull/3664

Reply:
Ok.